src/main/scala/Configs.scala

```diff
@@ -315,6 +315,7 @@ class BaseConfig extends Config (
           dataBeats = innerDataBeats,
           dataBits = site(CacheBlockBytes) * 8)
       }
+      case BootROMFile => "./bootrom/bootrom.hex"
```

Comment:
Does this assume that Chisel is invoked in the root directory of the repo?  Maybe that's OK...